presto-main/src/main/java/io/prestosql/server/HttpRequestSessionContext.java

```diff
@@ -287,13 +292,7 @@ public boolean supportClientTransaction()
 
     private static Map<String, String> parseSessionHeaders(HttpServletRequest servletRequest)
     {
-        Map<String, String> sessionProperties = new HashMap<>();
-        for (String header : splitSessionHeader(servletRequest.getHeaders(PRESTO_SESSION))) {
-            List<String> nameValue = Splitter.on('=').limit(2).trimResults().splitToList(header);
-            assertRequest(nameValue.size() == 2, "Invalid %s header", PRESTO_SESSION);
-            sessionProperties.put(nameValue.get(0), nameValue.get(1));
-        }
-        return sessionProperties;
+        return parseProperty(servletRequest, PRESTO_SESSION);
```

Comment:
can you please extract this refactor as separate commit

Reply:
In general, I agree, but might be overkill for a single 5 line method. Either way is fine with me.